app/Resources/views/modules/header.html.twig

```diff
@@ -37,7 +39,7 @@
                         </ul>
                     </li>
                     {% else %}
-                    <li class="navbar-menu"><a class="navbar-link" href="{{ path('member_connection') }}">{{ 'connection' |trans |upper}}</a></li>
+                    <button id="connexionButton" type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#registerLoginModal">{{ 'connection' |trans |upper}}</button>
```

Comment:
I think the style on this button seems pretty cheap, the borders are not great, we should remove them.
In order to do so, first remove the btn-info class here